packages/home-spa/.env.example

```diff
@@ -6,3 +6,4 @@ OP_SUBSCRIPTIONS_URL=
 
 ## SENTRY DSN
 # SENTRY_DSN=
+# Checking changed files
```

Comment:
Can you remove this line?
Looks like something that was used to test the action job. Not something that should be merged.

Reply:
Needed for testing after merge is done